ob_v3p0/impl/extensions.md

```diff
@@ -0,0 +1,1353 @@
+## Open Badges Extensions
+
+[Open Badges 3.0](http://www.imsglobal.org/spec/ob/v3p0#extending) and [Comprehensive Learner Record 2.0](http://www.imsglobal.org/spec/clr/v2p0#extending) allows extensibility in several ways: data model, extensible enumerated vocabularies and API.
+
+Open Badges 2.0 extensions are community developed contributions to the Open Badges Specification. Althought these extensions are not part of the core specification, 1Edtech keeps a list of community extensions at https://www.imsglobal.org/sites/default/files/Badges/OBv2p0Final/extensions/index.html#CommunityExtensions.
+
+These extension are compatible with the new version of Open Badges and CLR with some minor changes. Precisely, they are an extension of the data model. However, some of these extensions are deprecated in favor of the spec itself.
```

Comment:
I think this paragraph is confusing throughout. Would it be reasonable to not make any reference to the 2.0 extension and just to link to or include specific 3.0 compatible versions and guidance?